docs/docs/user-guide/cloud-storage.mdx

```diff
@@ -62,6 +62,19 @@ Rasa supports using [S3](https://aws.amazon.com/s3/) ,
 
   If there is no container with the name `AZURE_CONTAINER`, Rasa will create it.
 
+* Other Remote Storage
+
+  Support for other remote storages can be achieved by using an external library or implementing
+  a custom module for it by extending the `rasa.nlu.persistor.Persistor` class.
+
+  Start the Rasa server with ``remote-storage`` option set to the module path of model persistor.
+
+  A single command is:
+
+  ```bash
+  rasa run --remote-storage rasa.nlu.persistor.Persistor
```

Comment:
That should be a custom module. People might misunderstand if we give the parent class as example here.
```suggestion
  rasa run --remote-storage <your module>.<class name>
```